README.md

```diff
@@ -21,22 +37,22 @@ To run the local development server, first clone this repo.
 Install dependencies:
 
 ```bash
-yarn
+yarn i
```

Comment:
@steveruizok we use `yarn` here but use `npm` above - we should prbly be consistent - otherwise it'll be confusing - which one should we go with?

Reply:
actually, we should specify `yarn` - a lot of our tooling/docs assumes yarn